### PR TITLE
feature(auction) genesis

### PR DESCRIPTION
**Pull request by robert-zaremba**

### CODE REVIEW

File: x/auction/genesis.go [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
 // DefaultGenesis creates a default genesis state
 func DefaultGenesis() *GenesisState {
-	// TODO
-	return &GenesisState{}
+	return &GenesisState{
+		RewardsParams:   RewardsParams{BidDuration: 14 * 24 * 3600}, // 14 days
```
Comment (gsk967) [inline x/auction/genesis.go:13]:
```suggestion
		RewardsParams:   RewardsParams{BidDuration:  time.Second * 14 * 24 * 3600}, // 14 days
```

**Review by gsk967 — APPROVED** [review]
utACK, 
Small suggestion , can you add 
```go
func nextRewardsID(currentId int) int {
...
}

```

File: x/auction/genesis.go (hunk 1 above)
Comment (robert-zaremba) [reply x/auction/genesis.go:13]: This is correct, see the params docs: 
> // bid_duration is duration of the bid phase in seconds.

Comment (gsk967) [reply x/auction/genesis.go:13]: Okay

**Pull request merged**